.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,116 @@
+name: Publish xrpl-py 🐍 distribution 📦 to PyPI
+on:
+  push:
+    tags:
+        - 'v*' # Run on version tags only
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+    env:
+        POETRY_VERSION: 1.8.3
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ">=3.8,<3.13"
```

Comment:
Don’t we have a file to specify the version of Python?

Reply:
I haven't come across such a file. We specify all the supported versions of python explicitly in a list, here is an example: https://github.com/XRPLF/xrpl-py/blob/edfb99b5f73d9535677af1243e46c02566e659b6/.github/workflows/integration_test.yml#L20

But its a good idea to refactor these common elements from all the Github Actions workflows. I need to investigate how to link external files/ENV-VARIABLES inside these workflows.